### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by sidrb-msft**

### CODE REVIEW

**Review by samriddhi-MSFT — APPROVED** [review]
The changes look good.
I am not sure how to be able to merge the changes though.

**Comment by illfated** [comment]
Looks like the document needs to be updated to have someone on the current team named for automatic assignment. 
@eross-msft , do you happen to know to whom this document assignment should be?

File: EssentialsDocs/get-started/release-notes.md [hunk 1]
```diff
@@ -23,3 +23,8 @@ Before you connect a client computer to the server, make sure that you open Wind
 
 ### Client computer restore may not succeed after migration to Windows 10
  After updating a client computer from Windows 8.1 to Windows 10, client computer restore may not function correctly. You'll need to reconnect the client to the Windows Server 2016 Essentials.
+
+## Enabling TLS 1.2 for Windows Server Essentials
+In order to comply with the latest compliance standards for the Federal Risk and Authorization Management Program (FedRAMP), Microsoft plans to disable Transport Layer Security (TLS) versions 1.1 and 1.0 and enable TLS 1.2 in Windows Server Essentials. This would be done for both Client and Server computers.
+
+Please follow the steps mentioned in this released communication : https://docs.microsoft.com/en-us/mem/configmgr/core/plan-design/security/enable-tls-1-2-client
```
Comment (JasonGerend) [inline EssentialsDocs/get-started/release-notes.md:30]:
```suggestion
To do enable TLS 1.2, see [How to enable TLS 1.2 on clients](/mem/configmgr/core/plan-design/security/enable-tls-1-2-client).
```

**Comment by PRMerger12** [comment]
@JasonGerend : Thanks for your contribution! The author(s) have been notified to review your proposed change. @nnamuhcs

**Review by JasonGerend — CHANGES_REQUESTED** [review]
A couple little edits then this should be ready to publish.

**Comment by PRMerger17** [comment]
@sidrb-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change. @nnamuhcs

**Comment by PRMerger6** [comment]
@JasonGerend : Thanks for your contribution! The author(s) have been notified to review your proposed change. @nnamuhcs

**Review by JasonGerend — APPROVED** [review]

**Comment by JasonGerend** [comment]
#sign-off

**Pull request merged**